doc/release-notes.md

```diff
@@ -93,6 +93,9 @@ New settings
 Tools and Utilities
 -------------------
 
+- A new `-loglevel` configuration option allows setting global and per-category
+  log severity levels. (#25614)
```

Comment:
Shouldn't this be in a newly added file `doc/release-notes-25614.md`. From developer-notes:

```
Release notes should be added to a PR-specific release note file at
`/doc/release-notes-<PR number>.md` to avoid conflicts between multiple PRs.
```

Reply:
A merge conflict looks unlikely here (and this guideline seems rarely followed nowadays).

Reply:
Ok, I am fine with this either way.

In general, if a guideline is rarely followed then it is better to remove it from the guidelines, rather than keep breaking it.

Reply:
I am not sure what to do here myself, but maybe it is best to remove the doc/release-notes.md file from the repo to enforce the guideline? (There is still the `doc/release-notes-empty-template.md` file)

Reply:
IIRC the guideline was being followed but then it began being suggested to reviewers not to create a separate file when a merge conflict seemed unlikely.  I have a WIP branch to update the release notes (https://github.com/jonatack/bitcoin/commits/2022-08-developer-notes-updates) that includes a change like this.

```diff
- Release notes should be added to a PR-specific release note file at
- `/doc/release-notes-<PR number>.md` to avoid conflicts between multiple PRs.
- All `release-notes*` files are merged into a single
+ When it seems likely to cause a merge conflict, a release note should be added
+ in a PR-specific file named `/doc/release-notes-<PR number>.md`.  These
+ `release-notes*` files are merged into a single
[/doc/release-notes.md](/doc/release-notes.md) file prior to the release.
``` 

No strong opinion as to whether we want to continue in this direction, I haven't merged the release notes yet so don't know the pain level involved in doing that. 

(This pull no longer has a release note.)